src/index.html

```diff
@@ -9,6 +9,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navbar">
+      <a class="logo nav__link" href="#">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" data-qa="hover"
+            href="#">laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">gadgets</a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
          </li>
          
          <li class="nav__item">
```
You forgot to add blank line here